### PR TITLE
Generate schools jsons

### PR DESCRIPTION
**Pull request by carmelp16**

### CODE REVIEW

File: anyway/parsers/injured_around_schools_2022.py [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+)
+from anyway.utilities import time_delta, chunks
+from anyway.app_and_db import db
+
+class SchoolsJsonFile(Enum):
+    MAIN_FILE = ""
+
+SUBTYPE_ACCIDENT_WITH_PEDESTRIAN = 1
+LOCATION_ACCURACY_PRECISE = True
+LOCATION_ACCURACY_PRECISE_INT = 1
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:33]: I suggest adding LOCATION_ACCURACY_PRECISE_LIST = [1, 3, 4]

File: anyway/parsers/injured_around_schools_2022.py [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+        .filter(InvolvedMarkerView.geom.intersects(pol_str))
+        .filter(
+            or_(
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_1_CODE),
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_3_CODE),
+            )
+        )
+        .filter(InvolvedMarkerView.accident_timestamp >= start_date)
+        .filter(InvolvedMarkerView.accident_timestamp < end_date)
+        .filter(InvolvedMarkerView.location_accuracy == LOCATION_ACCURACY_PRECISE_INT)
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:85]: And then correct this line to `.filter(InvolvedMarkerView.location_accuracy.in_(LOCATION_ACCURACY_PRECISE_LIST))`

File: anyway/parsers/injured_around_schools_2022.py [hunk 3]
```diff
@@ -0,0 +1,560 @@
...
+        db.session.query(InvolvedMarkerView)
+        .filter(InvolvedMarkerView.geom.intersects(pol_str))
+        .filter(
+            or_(
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_1_CODE),
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_3_CODE),
+            )
+        )
+        .filter(InvolvedMarkerView.accident_timestamp >= start_date)
+        .filter(InvolvedMarkerView.accident_timestamp < end_date)
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:87]: @carmelp16 
i think we need <= here don't we?

File: anyway/parsers/injured_around_schools_2022.py [hunk 4]
```diff
@@ -0,0 +1,560 @@
...
+        .filter(InvolvedMarkerView.location_accuracy.in_(LOCATION_ACCURACY_PRECISE_LIST))
+        .filter(InvolvedMarkerView.age_group.in_(AGE_GROUPS))
+        .filter(InvolvedMarkerView.injury_severity.in_([1, 2, 3]))
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:90]: Can you make sure it's a list at the top and use a variable?

File: anyway/parsers/injured_around_schools_2022.py [hunk 5]
```diff
@@ -0,0 +1,560 @@
...
+        .filter(
+            or_(
+                (InvolvedMarkerView.injured_type.in_(INJURED_TYPES)),
+                (InvolvedMarkerView.involve_vehicle_type.in_(VEHICLE_TYPES))
+            )
+        )
+        .filter(
+            InvolvedMarkerView.accident_hour_raw.between(SEVEN_AM_RAW, SEVEN_PM_RAW)
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:98]: @carmelp16 just wanna make sure is this between inclusive for both sides?

File: anyway/parsers/injured_around_schools_2022.py [hunk 6]
```diff
@@ -0,0 +1,560 @@
...
+        location_accurate = 1
+        location_approx = 1
+
+    ui_url_map_only = ANYWAY_UI_FORMAT_MAP_ONLY.format(
+        latitude=latitude,
+        longitude=longitude,
+        start_date=start_date.strftime(DATE_URL_FORMAT),
+        end_date=end_date.strftime(DATE_URL_FORMAT),
+        acc_type=SUBTYPE_ACCIDENT_WITH_PEDESTRIAN,
+        location_accurate=location_accurate,
```
Comment (atalyaalon) [inline anyway/parsers/injured_around_schools_2022.py:136]: Here we actually changed the logic so we want to make sure the links will be aligned with it as well.
I suggest the following:
- remove location accuracy from the links
- Filter in `models.py` where light_transportation == True (line 613)

File: anyway/parsers/injured_around_schools_2022.py (hunk 3 above)
Comment (carmelp16) [reply anyway/parsers/injured_around_schools_2022.py:87]: @atalyaalon Yep you're right 👌🏽

File: anyway/parsers/injured_around_schools_2022.py (hunk 5 above)
Comment (carmelp16) [reply anyway/parsers/injured_around_schools_2022.py:98]: @atalyaalon Yes it's indeed inclusive:
https://www.w3schools.com/sql/sql_between.asp

**Pull request closed without merging**